static/scenes/railroad/js/game.js

```diff
@@ -34,22 +35,14 @@ class Game {
     this.renderer.shadowMap.enabled = true;
     container.appendChild(this.renderer.domElement);
 
+    await app.Scene.preload();
+    await app.Present.preload();
```

Comment:
Optional: Here you could also do:

await Promise.all([promise1, promise2)];

Reply:
Good call, done.